config/manager/manager.yaml

```diff
@@ -48,8 +50,8 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 30Mi
+            memory: 100Mi
```

Comment:
If we're still having issues with speed, we might try giving it more than 1/10th of one CPU.